src/utils/frontend/FormSubmissionHelper.test.js

```diff
@@ -19,7 +19,7 @@ describe('FormSubmissionHelper', async () => {
   });
 
   describe('Validations', () => {
-    vi.mock('./session-core', () => ({
+    vi.mock('../solid/session-core', () => ({
```

Comment:
Need to update the path of these module mocks. These mocks basically say "I want the `FormSubmissionHelper` to call session-core, but I don't actually care what session-core does."

Reply:
Saw the updates, thanks for going over them with me on the voice chat!